concordium-sdk/src/main/java/com/concordium/sdk/responses/blockinfo/BlockInfo.java

```diff
@@ -0,0 +1,100 @@
+package com.concordium.sdk.responses.blockinfo;
+
+import com.concordium.sdk.responses.accountinfo.AccountInfo;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.Hash;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import lombok.*;
+
+import java.time.OffsetDateTime;
+
+@ToString
+@Data
+@RequiredArgsConstructor
+@Getter
+public class BlockInfo {
+    /**
+     * / Hash of the block.
+     */
+    private Hash blockHash;
+    /**
+     * / The total energy consumption of transactions in the block.
+     */
+    private Integer transactionEnergyCost;
+    /**
+     * / Identity of the baker of the block. For non-genesis blocks the value is
+     * / non-null.
+     */
+    private Integer blockBaker;
+    /**
+     * / Hash of the block state at the end of the given block.
+     */
+    private Hash blockStateHash;
+    /**
+     * / Slot time of the slot the block is in. In contrast to
+     * / {@link BlockInfo#blockArriveTime} this is an objective value, all nodes
+     * / agree on it.
+     */
+    private OffsetDateTime blockSlotTime;
+    /**
+     * / Parent block pointer.
+     */
+    private Hash blockParent;
+    /**
+     * / Time when the block was first received by the node. This can be in
+     * / principle quite different from the arrive time if, e.g., block execution
+     * / takes a long time, or the block must wait for the arrival of its parent.
```

Comment:
```suggestion
     *  Time when the block was first received by the node. This can be in
     *  principle quite different from the arrive time if, e.g., block execution
     *  takes a long time, or the block must wait for the arrival of its parent.
```